css/freebusy.scss

```diff
@@ -28,7 +28,8 @@
 	}
 
 	.blocking-event-free-busy {
-		border-color: red;
+		border-radius: var(--border-radius);
+		border-color: var(--color-primary-element);
```

Comment:
![Bildschirmfoto vom 2024-02-27 11-55-52](https://github.com/nextcloud/calendar/assets/1374172/900d21ba-1cce-4a6f-b8b1-ee1f6c9cde61)

fine-tune this for the --first-row and --last-row only